lib/radioactive_bunnies/worker.rb

```diff
@@ -1,17 +1,14 @@
 require 'atomic'
 require 'radioactive_bunnies/context'
-
+require 'radioactive_bunnies/deadletter_worker'
 module RadioactiveBunnies::Worker
 
-  def ack!
-    true
-  end
-
   def work
   end
 
   def self.included(base)
     base.extend ClassMethods
+    base.extend RadioactiveBunnies::DeadletterWorker::ClassMethods
```

Comment:
Well we could fire this mix-in during the from_queue method, I'd have to dig around a bit.  This isn't much bloat, but let me know if you think it's too much?